Pipeline.md

```diff
@@ -0,0 +1,40 @@
+# Pipeline
+
+Coconut’s pipe operators allow for pipeline-style function application, which makes multiple function calls more readable.
+ 
+The different operators are:
+```python
+(|>)    => pipe forward
+(|*>)   => multiple-argument pipe forward
+(<|)    => pipe backward
+(<*|)   => multiple-argument pipe backward
+```
+*Note: refer to the [documentation](http://coconut.readthedocs.io/en/master/DOCS.html#operators) for information about operator precedence and associativity.*
```

Comment:
I think since we're putting this forward, we might as well do four short examples, one with each operator just to get people familiar with them.

Reply:
Can confirm, especially because I didn't particularly understand what they meant in the original documentation until I saw the examples of use later.